server/vulnerabilities/nvd/sync/cve_syncer_test.go

```diff
@@ -0,0 +1,145 @@
+package nvdsync
```

Comment:
The test doesn't cover initial sync or update functions. Anything we can do to test those?

Reply:
I will write tests on Wed to increase coverage.